.github/workflows/docker_image.yml

```diff
@@ -322,7 +320,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        digests: [ vcpkg-centos-8, vcpkg-centos-9, centos-8-jdk8, centos-8-jdk11, centos-8-jdk17, centos-9-jdk8, centos-9-jdk11, centos-9-jdk17 ]
+        digests: [ vcpkg-centos-8, centos-8-jdk8, centos-8-jdk17 ]
```

Comment:
We only push these three images, together with 1.5-vcpkg-centos-7 and 1.5-centos-9-jdk8-cudf, to docker hub. They are required to help verify PR target for branch 1.5.